### PR TITLE
Fresher preset updates - Round 2 ;)

### PR DESCRIPTION
**Pull request by SupaflyFPV**
Hi All

Ok here goes...RC LINK stuff now removed...pls take a look thanks

@limonspb - can I add the 'Firmware Version 4.4' also to the description?

### CODE REVIEW

**Comment by limonspb** [comment]
> Hi All
> 
> Ok here goes...RC LINK stuff now removed...pls take a look thanks
> 
> @limonspb - can I add the 'Firmware Version 4.4' also to the description?

i think for 4.4 we will have to make a whole new folder and new presets

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt [hunk 1]
```diff
@@ -1,36 +1,68 @@
...
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: 
 #$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
 #$ INCLUDE: presets/4.3/tune/defaults.txt
 #$ INCLUDE: presets/4.3/filters/defaults.txt
 
+# -- Logging --
+
+set blackbox_sample_rate = 1/4
+set debug_mode = GYRO_SCALED
```
Comment (limonspb) [inline presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:52]: plz remove

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt [hunk 2]
```diff
@@ -1,36 +1,68 @@
...
 #$ DESCRIPTION: Enjoy!
-#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/124
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: Support, feedback and chat:
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION:*[FACEBOOK SUPPORT/FEEDBACK](https://www.facebook.com/groups/supaflyfpv/)
```
Comment (limonspb) [inline presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:37]: if you want to kave a list of items here, then should be white spaces around the *

**Comment by SupaflyFPV** [comment]
> > Hi All
> > Ok here goes...RC LINK stuff now removed...pls take a look thanks
> > @limonspb - can I add the 'Firmware Version 4.4' also to the description?
> 
> i think for 4.4 we will have to make a whole new folder and new presets. CLI commands for tuning will be somewhat different.

ok but the development master branch is currently called 4.4 so was wondering how that works...and also there is the warning for agree, no continue....

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt [hunk 3]
```diff
@@ -1,36 +1,68 @@
...
+#$ FIRMWARE_VERSION: 4.3, 4.4
 #$ CATEGORY: TUNE
 #$ STATUS: OFFICIAL
 #$ KEYWORDS: Supafly, SupaflyFPV, 4S, 6S, Freestyle, 5 inch, 5"
 #$ AUTHOR: SupaflyFPV
-#$ DESCRIPTION: SupaflyFPV Community Freestyle Preset for 5 Inch Builds
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION: [<img src="https://supaflyfpv.com/wp-content/uploads/2022/06/Logo-Cropped192.svg" width="220px" style="margin-left: auto; margin-right: auto; display: block;"/>](https://supaflyfpv.com)
```
Comment (limonspb) [inline presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:10]: to have a link to the image you should do something like that:
https://api.betaflight.com/firmware-presets/presets/4.3/tune/UAV_tech_6-7in.txt

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt [hunk 4]
```diff
@@ -1,36 +1,68 @@
...
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
 #$ DESCRIPTION:
-#$ DESCRIPTION: Select the option for your build with or without HD camera (eg GoPro) as the baseline.
+#$ DESCRIPTION: ### SupaflyFPV 5 Inch Freestyle Preset
 #$ DESCRIPTION:
-#$ DESCRIPTION: Half to one click up or down Master Slider to tweak the tune - in OSD or Configurator.
+#$ DESCRIPTION: <br>
```
Comment (limonspb) [inline presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:16]: no need BR in the middle of non-html content
for empty lines just put empty #$ DESCRIPTION:

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt [hunk 5]
```diff
@@ -1,36 +1,68 @@
...
 #$ DESCRIPTION:
-#$ DESCRIPTION: For a Master Slider setting tailored to your Lipo and Motor Kv go here: https://tinyurl.com/SupaflyFPV
+#$ DESCRIPTION: #### New Higher Performance! Pick your options from the drop-down:
 #$ DESCRIPTION:
-#$ DESCRIPTION: RPM Filtering for best performance (default) - or de-select for Non RPM Setup (check esc compatibility).
+#$ DESCRIPTION: <br>
```
Comment (limonspb) [inline presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:20]: remove br plz

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt [hunk 6]
```diff
@@ -1,36 +1,68 @@
...
 #$ DESCRIPTION:
-#$ DESCRIPTION: If using F7/H7 Flight Controller run 8k Pidloop and turn off Gyro Low Pass Filter 2 for max performance.
+#$ DESCRIPTION: * RPM Filtering recommended on Blheli32 esc or de-select if ESC is not compatible
+#$ DESCRIPTION: * Apply 'Rough Motors' option if your motors or frame are old/beat up
+#$ DESCRIPTION: * Pick starter PIDs option - 4s works for 2400-2700kv motors or 6s for 1750 - 1950kv
+#$ DESCRIPTION: * For further optimisation to your specific motor kv [use this custom PID calculator](https://supaflyfpv.com/betaflight-setup/)
+#$ DESCRIPTION: 
+#$ DESCRIPTION: <br>
```
Comment (limonspb) [inline presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:27]: remove BR plz

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt [hunk 7]
```diff
@@ -1,36 +1,68 @@
...
 #$ DESCRIPTION:
 #$ DESCRIPTION: Enjoy!
-#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/124
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
```
Comment (limonspb) [inline presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:31]: remove BR plz

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt [hunk 8]
```diff
@@ -1,36 +1,68 @@
...
+#$ DESCRIPTION:
+#$ DESCRIPTION: Support, feedback and chat:
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION:*[FACEBOOK SUPPORT/FEEDBACK](https://www.facebook.com/groups/supaflyfpv/)
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
```
Comment (limonspb) [inline presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:39]: remove BR plz

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt [hunk 9]
```diff
@@ -1,36 +1,68 @@
...
+#$ DESCRIPTION:
+#$ DESCRIPTION:*[DISCORD SUPPORT/FEEDBACK](https://discord.gg/Gs5nTx4rE2)
+#$ DESCRIPTION:
+#$ DESCRIPTION: <br>
```
Comment (limonspb) [inline presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:43]: remove BR plz

**Review by limonspb — CHANGES_REQUESTED** [review]
... and with the big complicated updates, i would recommend doing them for 1 file per PR only

**Comment by limonspb** [comment]
> > > Hi All
> > > Ok here goes...RC LINK stuff now removed...pls take a look thanks
> > > @limonspb - can I add the 'Firmware Version 4.4' also to the description?
> > 
> > 
> > i think for 4.4 we will have to make a whole new folder and new presets. CLI commands for tuning will be somewhat different.
> 
> ok but the development master branch is currently called 4.4 so was wondering how that works...and also there is the warning for agree, no continue....shall I remove the reference to 4.4 then?

agree means continue :) we need to change that word in the configurator.

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt (hunk 3 above)
Comment (SupaflyFPV) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:10]: ok...on recommendation - I copied the formatting as used by:

https://github.com/betaflight/firmware-presets/blob/master/presets/4.3/rates/mouseFreestyle.txt

Comment (limonspb) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:10]: in mousefpv preset there is no link on the image.

**Comment by limonspb** [comment]
@SupaflyFPV  please make it a separate PR for a single file first.
will be easier for me and for you.
And plz keep 1 commit :)

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt (hunk 3 above)
Comment (SupaflyFPV) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:10]: ok re link image change.
 re BR - I put the spaces in (like MouseFPV) in order to space the text. Left 5" is without, 6" is with:

Screenshot 2022-07-01 at 23 10 26

**Comment by SupaflyFPV** [comment]
> ... and with the big complicated updates, i would recommend doing them for 1 file per PR only

ok I will split per 'tune' the PR

**Comment by limonspb** [comment]
> > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> 
> ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....



> > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> 
> ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....

it's cuz everyone listened to @spatzengr and started using GitHub desktop )) Need a new video of how to squash commits in Desktop and fix your presets PR if you ever messed it up.

My recommendation - git bash :)

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt (hunk 3 above)
Comment (limonspb) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:10]: @SupaflyFPV  send me your branch name i will check with configurator + chrome dev tools

**Comment by SupaflyFPV** [comment]
> > > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> > 
> > 
> > ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....
> 
> > > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> > 
> > 
> > ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....
> 
> it's cuz everyone listened to @spatzengr and started using GitHub desktop )) Need a new video of how to squash commits in Desktop and fix your presets PR if you ever messed it up.
> 
> My recommendation - git bash :)

yeah it probably is that...I tried to remember what the commands were when we discussed it but couldn't...its merge issues - cannot then squash...if you can lay out common command sequences that would help a lot and avoid you repeating maybe...

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt (hunk 3 above)
Comment (SupaflyFPV) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:10]: https://github.com/SupaflyFPV/firmware-presets/tree/Fresher-Preset-Updates

Comment (limonspb) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:10]: I am so confused why this one works:

`[](https://supaflyfpv.com)`

and confused why <br> needed. I guess MARKDOWN eats empy lines.

So i guess yes, you need <br/> if you need extra spacing.

Plz make it <br/> tho. 

Sorry for the confusion

**Comment by spatzengr** [comment]
> > > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> > 
> > 
> > ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....
> 
> > > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> > 
> > 
> > ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....
> 
> it's cuz everyone listened to @spatzengr and started using GitHub desktop )) Need a new video of how to squash commits in Desktop and fix your presets PR if you ever messed it up.
> 
> My recommendation - git bash :)

Itnis easy to squash in the latest versions of Git Desktop.  Just grab multiple commits and right-click to select squash.  It then flips to automatically do a Force Push.

**Comment by sugaarK** [comment]
> > > > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> > > 
> > > 
> > > ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....
> > 
> > 
> > > > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> > > 
> > > 
> > > ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....
> > 
> > 
> > it's cuz everyone listened to @spatzengr and started using GitHub desktop )) Need a new video of how to squash commits in Desktop and fix your presets PR if you ever messed it up.
> > My recommendation - git bash :)
> 
> Itnis easy to squash in the latest versions of Git Desktop. Just grab multiple commits and right-click to select squash. It then flips to automatically do a Force Push.

Lols @limonspb hates for desktop

**Comment by SupaflyFPV** [comment]
> > > > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> > > 
> > > 
> > > ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....
> > 
> > 
> > > > ... and with the big complicated updates, i would recommend doing them for 1 file per PR only
> > > 
> > > 
> > > ok I will split per 'tune' the PR. I am trying with the commits but sometimes there is squash problem....
> > 
> > 
> > it's cuz everyone listened to @spatzengr and started using GitHub desktop )) Need a new video of how to squash commits in Desktop and fix your presets PR if you ever messed it up.
> > My recommendation - git bash :)
> 
> Itnis easy to squash in the latest versions of Git Desktop. Just grab multiple commits and right-click to select squash. It then flips to automatically do a Force Push.

I will grab the latest version hopefully that avoids the tangles. Otherwise I like using it.

File: presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt (hunk 1 above)
Comment (SupaflyFPV) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:36]: this is useful for troubleshooting down the line should it be necessary - why the need to remove?

Comment (limonspb) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:36]: A person might have his blackbox already setup, or he simply does not want it.

The general rule, we can't set parameters that are not part of the tune/filters outside of the unchecked options.

Comment (SupaflyFPV) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:36]: Ok, the categorisation issue...how about some debug presets? If so what category should they go in...

Comment (limonspb) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:36]: we already have some:
https://github.com/betaflight/firmware-presets/blob/master/presets/4.3/other/blackbox_disable.txt

Comment (SupaflyFPV) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt:36]: ok but I think I'll make a troubleshoot preset or something...that just disables parameters for more space...

Comment (sugaarK) [reply presets/4.3/tune/SupaflyFPV_Freestyle_5_Inch_EasyTune.txt]: It should be set as unchecked an option as it fall out side of what a tune is

**Comment by sugaarK** [comment]
@SupaflyFPV  what we doing wit this

**Comment by SupaflyFPV** [comment]
> @SupaflyFPV what we doing wit this

changes are ready to go, just need to arrange a few things before sorting it

**Pull request closed without merging**